x/evm/keeper/sdk_tx.go

```diff
@@ -0,0 +1,57 @@
+package keeper
+
+import (
+	"fmt"
+	"math"
+
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/evmos/ethermint/x/evm/types"
+)
+
+var DefaultPriorityReduction = sdkmath.NewIntFromUint64(1000000)
```

Comment:
Because the `priority` is a `int64` while token could be very big, similar to the rationale of `DefaultPowerReduction`.

Reply:
added comments:
```
// DefaultPriorityReduction is the default amount of price values required for 1 unit of priority.
// Because priority is `int64` while price is `big.Int`, it's necessary to scale down the range to keep it more pratical.
// The default value is the same as the `sdk.DefaultPowerReduction`.
```

Reply:
let's use `var DefaultPriorityReduction = sdk.DefaultPowerReduction` instead

Reply:
this var is moved to another PR.